src/pages/teachers/programmes/[programmeSlug]/units.tsx

```diff
@@ -72,20 +84,25 @@ const UnitListingPage: NextPage<UnitListingPageProps> = ({
 
   const router = useRouter();
   const themeSlug = router.query["learning-theme"]?.toString();
+  const categorySlug = router.query["category"]?.toString();
+  const yearGroupSlug = router.query["year"]?.toString();
+  const [selectedCategory, setSelectedCategory] = useState<string | null>(null);
+  const [skipFiltersButton, setSkipFiltersButton] = useState(false);
 
   const [selectedThemeSlug, setSelectedThemeSlug] = useState<
     string | undefined
   >(themeSlug);
 
-  const unitsFilteredByLearningTheme = filterLearningTheme(
-    selectedThemeSlug,
+  const filteredUnits = filterUnits({
+    themeSlug: selectedThemeSlug,
+    categorySlug,
```

Comment:
should this be using `selectedCategory`?

Reply:
Removed, not being utilised any more 👍